### PR TITLE
fix: fixed description not being long enough in embeds

### PR DESCRIPTION
**Pull request by Ahmadh26**
add a check if description length is not bigger than 0, then use "No description"

a fix for issue #78

### CODE REVIEW

File: 2-starter-with-api/src/commands/slashes.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export class SlashExample {
         .setFooter({ text: `Page ${i + 1} of ${commands.length}` })
         .setTitle("**Slash command info**")
         .addFields({ name: "Name", value: cmd.name })
-        .addFields({ name: "Description", value: cmd.description });
+        .addFields({ name: "Description",  value: cmd.description.length > 0 ? cmd.description : "No description"  });
```
Comment (VictoriqueMoe) [inline 2-starter-with-api/src/commands/slashes.ts:23]: Add nullish checks

File: 1-starter/src/commands/slashes.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ export class SlashExample {
         .setFooter({ text: `Page ${i + 1} of ${commands.length}` })
         .setTitle("**Slash command info**")
         .addFields({ name: "Name", value: cmd.name })
-        .addFields({ name: "Description", value: cmd.description });
+        .addFields({ name: "Description", value: cmd.description.length > 0 ? cmd.description : "No description" });
```
Comment (VictoriqueMoe) [inline 1-starter/src/commands/slashes.ts:23]: Again, add nullish checks

**Review by VictoriqueMoe — CHANGES_REQUESTED** [review]

File: 2-starter-with-api/src/commands/slashes.ts (hunk 1 above)
Comment (samarmeena) [reply 2-starter-with-api/src/commands/slashes.ts:23]: why?

**Comment by samarmeena** [comment]
fix prettier errors

**Pull request merged**